### PR TITLE
V2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: tests.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    # -------------------------------
+
+    args.with_sftp = "1"
+	args.phpversion = "7.4"
+	args.with_mysql = "1"
+
+[7.3_test]
+
+    only = ["install.subdir", "backup_restore", "upgrade" ]
+    args.phpversion = "7.3"
```
Comment (alexAubin) [inline tests.toml:16]: Uuuh unfortunately we can't use `.` in the name here, though it acts as if it works, but this is equivalent to write:

```
[7]
    [7.3_test]
            only = ["install.subdir", "backup_restore", "upgrade" ]
```

with `7` being one level, and `3_test` being another level

which is not what we want :sweat_smile: 

Maybe we should rename the test to `php_73` for example or `73_test` or whatever, but without a dot :D

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by tituspijean** [comment]
Weirdly, the bot did not answer. Here are the badge and summary:

[Test Badge](https://ci-apps-dev.yunohost.org/ci/job/6605)
[](https://ci-apps-dev.yunohost.org/ci/job/6605)

File: doc/POST_INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+If you have requested a MYSQL database, please find information about this SQL database:
+
+Database user: __DB_USER__
+Database name: __DB_NAME__
+Password: __DB_PWD__
+
+You can connect to this repository by using SFTP with the following credentials.
+Domain: __DOMAIN__
+Port: __SSH_PORT__
+User: __APP__
```
Comment (tituspijean) [inline doc/POST_INSTALL.md:10]:
```suggestion
User: __ID__
```

File: sources/www/index.html [hunk 3]
```diff
@@ -20,7 +20,7 @@ <h2>Edit this site</h2>
       <dt>Domain</dt>
       <dd>__DOMAIN__</dd>
       <dt>Port</dt>
-      <dd>22 (or the port you defined <a href="https://yunohost.org/security">if you changed the SSH port</a>)</dd>
+      <dd>__SSH_PORT__</dd>
       <dt>User</dt>
       <dd>__APP__</dd>
```
Comment (tituspijean) [inline sources/www/index.html:25]:
```suggestion
      <dd>__ID__</dd>
```

**Review by tituspijean — CHANGES_REQUESTED** [review]

**Comment by tituspijean** [comment]
(rebooting job abobe ^)

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**